<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimePeriodChooser</class>
 <widget class="QWidget" name="TimePeriodChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>131</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="previousButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/resources/leftarrow.png</normaloff>:/new/prefix1/resources/leftarrow.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="weekButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Week</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="monthButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="nextButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/resources/rightarrow.png</normaloff>:/new/prefix1/resources/rightarrow.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="fromLabel">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="toLabel">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QDateEdit" name="beginDateChooser"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QDateEdit" name="endDateChooser"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
